recipe/conda_build_config.yaml

```diff
@@ -31,8 +31,9 @@ zip_keys:            # [linux]
   - spdlog           # [linux]
   - pyarrow          # [linux]
 # Because of the awkwardness required for the cloud variant above
-# (fmt9+spdlog1.11), we have to migrate the osx-* manually here. Once the
-# conda-forge-pinning-feedstock is updated to spdlog 1.13, we can remove the
-# manual pin below
+# (fmt9+spdlog1.11), we also have to manually pin fmt/spdlog for the osx-*
+# builds
+fmt:                 # [not linux]
+  - 11               # [not linux]
```

Comment:
The 2 lines above were the key. Because fmt is also undergoing a migration at the moment, we need to manually pin it for the osx-* builds